src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.html

```diff
@@ -0,0 +1,30 @@
+<div dropdown>
+  <a dropdownToggle
+     class="dropdown-toggle"
+     data-toggle="dropdown"
+     i18n-title
+     title="Logged in user">
+    <i class="fa fa-user fa-fw"></i>
+    <span class="caret"></span>
+  </a>
+  <ul *dropdownMenu
+      class="dropdown-menu dropdown-menu-right"
+      role="menu">
+    <li class="disabled"
+        role="menuitem">
+      <a i18n
+         class="dropdown-item disabled"
+         href="#">Signed in as
+        <strong>{{ username }}</strong>
+      </a>
+    </li>
+    <li class="divider dropdown-divider"></li>
+    <li role="menuitem">
+      <a class="dropdown-item"
+         (click)="logout()">
+        <i class="fa fa-sign-out fa-fw"></i>
+        <span i18n> Sign out</span>
```

Comment:
Is the white space before 'Sign' necessary?

Reply:
@familyuu can you please remove this unnecessary white space? After that we can merge this PR.